qemu/tests/cfg/migration_with_block.cfg

```diff
@@ -0,0 +1,103 @@
+    virt_test_type = qemu
+    type = migration_with_block
+    images += " stg0"
+    image_name_stg0 = images/storage0
+    image_size_stg0 = 40G
+    blk_extra_params_stg0 = "serial=TARGET_DISK0"
+    remove_image_stg0 = yes
+    force_create_image_stg0 = yes
+    kill_vm_on_error = yes
+    virtio_scsi:
+        driver_name = vioscsi
+    virtio_blk:
+        driver_name = viostor
+    variants:
+        - @default:
+            variants:
+                - shutdown_vm:
+                    shutdown_vm = yes
+                    command_qmp = system_powerdown
+                    Windows:
+                        command_shell = shutdown -t 0 -s
+                    Linux:
+                        command_shell = shutdown -h now
+                - reboot_vm:
+                    reboot_vm = yes
+                    command_qmp = system_reset
+                    Windows:
+                        command_shell = shutdown -t 0 -r
+                    Linux:
+                        command_shell = reboot
+            variants:
+                - send_qmp:
+                    command_opts = qmp,${command_qmp}
+                - send_shell:
+                    command_opts = shell,${command_shell}
+            variants:
+                - with_scsi_on2off:
+                    only virtio_blk
+                    src_addition_desc = 'with scsi=on'
+                    dst_addition_desc = 'with scsi=off'
+                    blk_extra_params_stg0 = "scsi=on,disable-legacy=off,disable-modern=on"
+                    set_dst_params = "{'blk_extra_params_stg0': 'scsi=off,disable-legacy=off,disable-modern=on'}"
+                - with_change_cdrom:
+                    src_addition_desc = 'with cdrom'
+                    dst_addition_desc = 'with new cdrom'
+                    only virtio_scsi
+                    with_cdrom = yes
+                    cdrom_orig_name = orig
+                    cdroms = ${cdrom_orig_name}
+                    image_boot_orig = yes
+                    cdrom_orig = /tmp/${cdrom_orig_name}.iso
+                    cdrom_orig_file = ${cdrom_orig}
+                    cdrom_new_name = new
+                    cdrom_new_file = /tmp/${cdrom_new_name}.iso
+                    pre_command_noncritical = no
+                    pre_command = 'dd if=/dev/zero of=/tmp/${cdrom_orig_name} bs=1M count=1024 && '
+                    pre_command += 'mkisofs -o /tmp/${cdrom_orig_name}.iso /tmp/${cdrom_orig_name} && '
+                    pre_command += 'dd if=/dev/zero of=/tmp/${cdrom_new_name} bs=1M count=2048 && '
+                    pre_command += 'mkisofs -o /tmp/${cdrom_new_name}.iso /tmp/${cdrom_new_name}'
+                    post_command = "rm -rf /tmp/${cdrom_new_name}.iso /tmp/${cdrom_new_name} "
+                    post_command += "/tmp/${cdrom_orig_name}.iso /tmp/${cdrom_orig_name}"
+                    check_items = '{"io-status": "ok", "device": "drive_${cdrom_orig_name}", "locked": False, "removable": True, '
```

Comment:
no need check "locked": False, "removable": True, '
need add check "tray_open": false,

Reply:
Because rhel8 and rhel7 show inconsistencies on the "locked" state,The upper level user will not use this, so delete it.

Reply:
The other options need to be check ? such as `check_items += '"iops_rd": 0, "iops_wr": 0, "ro": True, "drv": "raw", '`
or just ignore to check `"locked": False, "removable": True, '` and retains others.

Reply:
Regarding the second step of the case,  only need to check ‘"io-status": "ok", "device": "drive_${cdrom_orig_name}","tray_open": false, ’other parameters do not need to be detected, 
including: "locked": False, "removable": True, '"iops_rd": 0, "iops_wr": 0, "ro": Yes, "drv": "raw", "iops": 0, "bps_wr": 0, "encryption": False, "bps": 0, "bps_rd": 0, "type": " Unknown"

Reply:
> @kimi1978wy @peixiu
> This is my opinion with @XueqiangWei . Do you agree?

Yes, I agree.

Reply:
@zhenyzha that's ok for me